Calendar/CalLogHelpers/Invoke-GetMailbox.ps1

```diff
@@ -107,6 +107,10 @@ function CheckIdentities {
             Write-Host -ForegroundColor DarkRed "Mailbox [$Id] has CalendarVersionStoreDisabled set to True.  This mailbox will not have Calendar Logs."
             Write-Host -ForegroundColor DarkRed "Some logs will be available for Mailbox [$Id] but they will not be complete."
         }
+        if ($Account.RecipientTypeDetails -like "RoomMailbox" -or $Account.RecipientTypeDetails -like "EquipmentMailbox") {
```

Comment:
`-like` is being used actually like `-eq` here. 

![image](https://github.com/microsoft/CSS-Exchange/assets/22776718/70a82589-d965-4991-95b6-a360e44c50e1)